### DISCUSSION TITLE
I can't click "continue" while registering for global campus

### DISCUSSION THREAD

**Discussion by pushkargupta627** [community/community#60918] [GitHub Education]
### Hi everyone,

I am not able to press the continue button while registering the continue campus. I tried giving it the location access, and even tried doing it on different devices, but no luck. I have also selected all of the columns asked for. I am not able to understand why i am not able to continue. Here's the screenshot for the same...
<img width="483" alt="Screenshot 2023-07-15 at 4 30 06 PM" src="https://github.com/community/community/assets/75349029/b7788e97-d7c0-4ed1-a3d5-8b89c8e16c45">

**Comment by karakuscem** [comment]
If the e-mail address you chose is not the e-mail address given by your school, can you add the e-mail address given by your school and try? You can select it after adding the school e-mail address to your profile by clicking the Add an e-mail address button. I'm not sure if it's the solution but it's worth to try.

**Reply by pushkargupta627** [reply]
Well, I have already added and verified that, but then too its not working. Thanks by the way.

**Comment by samus-aran** [comment]
Hi @pushkargupta627 👋 

Are you able to confirm if you have successfully enabled your browser to read your location? Since settings for every browser are different, if you are having difficulty enabling us to see your location you should seek assistance from your browser's support pages. If you are still having difficulties enabling your location check out [this discussion on assessing your hardware or operating system's settings.](https://github.com/orgs/community/discussions/42246)

I would recommending you re-enable it, log out of GitHub and log back in before re-applying.

Let us know how you get along 👍

**Reply by pushkargupta627** [reply]
Thanks for your answer @samus-aran but I tried doing the same and it didn’t work out. I also tried doing so in a different browser with location enabled but then too the result was the same.
